x-pack/plugins/security_solution/public/timelines/components/timeline/body/custom_discover_grid.tsx

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+export const CustomDiscoverGrid = () => {
```

Comment:
Should this be part of this PR?

Reply:
Oops. Thanks for catching.. removed now.